src/main/java/edu/ksu/canvas/attendance/submitter/CanvasAssignmentAssistant.java

```diff
@@ -59,6 +59,7 @@ public void createAssignmentInCanvas(Long courseId, AttendanceAssignment attenda
 
         LOG.info("Created canvas assignment: " + canvasAssignmentOptional.get().getId());
         saveCanvasAssignmentId(courseId, canvasAssignmentOptional.get());
+        return canvasAssignmentOptional.get();
```

Comment:
The canvas assignment could be stored as a local instead of duplicating the get call.

It's a bit fuzzy on when to assign something to a local vs repeating a call. On one hand, repeating a call can be more readable. We are pretty removed from the days when we should assign to a local variable for performance reasons with compiler optimizations, caching, and better hardware. So I usually use my judgement when deciding instead of hard rules.

In this case, if saveCanvasAssignment is changed and returns empty for some reason, an exception will be thrown.